styles/style.styl

```diff
@@ -220,6 +220,46 @@ section:first-child
   .form--message
     background rgb(226, 33, 112)
 
+.channels
+  display: grid
+  grid-template-columns: 1fr 1fr 1fr
+  grid-gap: 1.1ex
+  justify-content: center
+  padding-inline-start: 0
+  // NOTE: Breakpoints could be improved in the future
+  @media screen and (max-width: 1000px)
+    grid-template-columns: 1fr 1fr
+  @media screen and (max-width: 700px)
+      grid-template-columns: 1fr
+
+.channel
+  vertical-align middle
+  border-radius 4px
+  list-style-type: none
+  margin: 0
+  padding: 0
+  border: .15ex solid gray
+  padding: 0
+  min-height: 70px
+  background-color: #fff
+  box-shadow: 0px 2px 5px 0px rgba(220, 213, 232, 0.60), inset 0px 0px 15px 0px rgba(220, 220, 220, 0.3)
+
+.channel__name
+  display: block
+  border-bottom: 1px solid gray
+  padding: 1ex
+  font-weight: bold
+  border-radius 4px
+  border-bottom-left-radius: 0
+  border-bottom-right-radius: 0
+  background-color: hsla(207, 73%, 97%, 1)
+  color: #111
```

Comment:
Why link and black text?

Reply:
Black text because it has larger contast than a blue link on top of blue background...